devtools/conda-envs/no_openeye.yaml

```diff
@@ -23,7 +23,7 @@ dependencies:
   - openff-utilities
   - openff-toolkit >=0.15
   - openff-forcefields >=2024.04.0
-  - openff-qcsubmit >=0.50
+  - openff-qcsubmit >=0.55
```

Comment:
Curious why you bumped it this far and not any more? Fine either way, just wanted to know if there was a reason I've forgotten about

Reply:
0.55 is the newest version of QCSubmit (I tried to be cute with the versioning and jumped to 0.50 when the MolSSI QC* packages did, but now it's just creating confusion since QCFractal/Portal are up to 0.58 and we're going at a different release pace)

Reply:
I intuit that 1 is the best tradeoff of effort and value, especially since the manual updates are somewhat documented

I don't think this case hits production, otherwise we'd need to think about pinning in packaging

Reply:
Thanks josh!

Reply:
Ahh, that makes sense. Thanks Josh!